src/lib/components/CourseLinks/index.js

```diff
@@ -116,11 +118,10 @@ class CourseLinks extends HTMLElement {
       Math.floor((newPages.length / children.length) * 10) * 10;
 
     // Fire analytics if there's been a 10%+ bucket jump.
-    if (ga && newPercent - oldPercent >= 10) {
-      ga('send', 'event', {
-        eventCategory: 'Course Events',
-        eventAction: `course: ${courseKey} progress`,
-        eventLabel: newPercent.toString(),
+    if (newPercent - oldPercent >= 10) {
+      logEvent(`course: ${courseKey} progress`, {
+        event_category: 'Course Events',
+        event_label: newPercent.toString(),
```

Comment:
Hmmm, this pattern of inlining some scripts makes this migration a lot more complex. The easiest option for this PR is probably to switch this back go using `gtag()` but I really don't like that as a long term solution.

The only good justification for inlining scripts is to ensure critical code can run as fast as possible. However, analytics reporting is clearly not critical code, so I'd argue this should be removed and added to the `src/lib/pages/course.js` script, which is bundled by rollup so it can include analytics exports.

Also, looking at the logic in the file, course progress is stored in `localStorage`, so it should be relatively easy for code in the `src/lib/pages/course.js` file to inspect `localStorage` and report progress.

I'll update this with a TODO.

/cc @devnook